### ISSUE TITLE
[BUG] Secondary click on folder closes app grid

### ISSUE THREAD

**Issue by meybonomme** [G-dH/vertical-workspaces#180]
This is not very expected behavior when a secondary click on a folder in the main app grid closes the application grid. Maybe the second click should also open the folder, because the folder doesn't have an options menu?

**Comment by G-dH** [comment]
This is actually related to the empty space click option.

**Comment by meybonomme** [comment]
> This is actually related to the empty space click option.

Yes, I know, but as you asked, I will make a separate issue for each bug or improvement suggestion I find.

I assume you can change this behavior?

**Comment by G-dH** [comment]
Hi! Are you still using V-Shell?

**Comment by meybonomme** [comment]
> Hi! Are you still using V-Shell?

Yes, I still use it and update whenever you make changes.

**Comment by G-dH** [comment]
Is this fix acceptable to you?

**Comment by meybonomme** [comment]
Based on the logic that the primary click opens and the secondary click closes, the current solution seems confusing. I think it would be more logical if the secondary click on a folder do nothing, like in the default GNOME shell.
